php-practice_1.php

```diff
@@ -1,41 +1,146 @@
 <?php
 // Q1 変数と文字列
-
+$name = '佐田';
+ $sentence = '私の名前は「' . $name . '」です。';
+   echo $sentence;
 
 // Q2 四則演算
-
+$num = 5 * 4;
+echo $num . "\n";
+echo $num / 2;
 
 // Q3 日付操作
-
+$message = "現在時刻は " . date('Y年m月d日H時i分s秒') . " です";
+echo $message;
 
 // Q4 条件分岐-1 if文
+$device = "windows"; 
+$os = "どちらでもありません";
 
+if ($device == "windows") {
+    $os = "windows";
+} else {
+    if ($device == "mac") {
+        $os = "mac";
+    } else {
+        $os = "どちらでもありません";
+    }
+}
 
-// Q5 条件分岐-2 三項演算子
+echo "使用OSは、$os です。";
 
+// Q5 条件分岐-2 三項演算子
+$age = 18;
+$message = ($age > 20 )? '未成年です。': '成人です。';
```

Comment:
[nits]
不要な空白まで適用されています！
```suggestion
$message = ($age > 20) ? '未成年です。': '成人です。';
```

Reply:
20の後ろの空白を消して）の後ろに空白つけました修正しました